practical/groovy/analyse_dataset_save_rois_and_summary_table.groovy

```diff
@@ -0,0 +1,298 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2018 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script shows how to analyse an OMERO dataset
+ * i.e. a collection of OMERO images.
+ * For that example, we use the analyse particles plugin.
+ * The generated ROIs are then saved back to OMERO.
+ * We create a summary CSV and a summary table of the measurement and attach
+ * them to the dataset.
```

Comment:
Two issues with this script here:


  1. The ``windowless=true`` somehow does not work for this script. If you run this script once, then close and reopen your Fiji, run this script second time -> the BF window will open and will break up the flow. Interestingly, the jython script of @will-moore has the ``windowless=true`` working in more robust manner, so that it actually can ``fix`` the behaviour of this script - just first run Will's script, then this script, and all will be fine :)

  2. I do not understand why the csv creation was removed. This is not what we intended for the workshop. In the workshop, we wanted to show the csv (albeit the one done previously). How can we claim now that we are showing the csv which was done by this script previously when this script does not produce any csv in the first place anymore ?

Reply:
2. I misunderstood. I will add the CSV file creation